smrf/tests/smrf_test_case.py

```diff
@@ -15,17 +17,71 @@ class SMRFTestCase(unittest.TestCase):
     The base test case for SMRF that will load in the configuration file
     and store as the base config. Also will remove the output
     directory upon tear down.
+
+    Runs the short simulation over reynolds mountain east
     """
-    dist_variables = [
+    DIST_VARIABLES = frozenset([
         'air_temp',
         'cloud_factor',
         'precip',
         'thermal',
         'vapor_pressure',
         'wind',
-    ]
+    ])
+
+    BASE_INI_FILE_NAME = 'config.ini'
+
+    test_dir = PurePath(smrf.__file__).parent.joinpath('tests')
```

Comment:
Per your preference, why not capitalize these?

Reply:
I kept this lower case to give it the same accessing syntax as you would do it by declaring a `@property` for an instance. All caps symbolizes a constant (to me). This can be argued as personal preference and I can also change it to use all caps.

What do you prefer?

Reply:
This works fine, it's not really a constant since it's dynamically set so makes sense to have it lower case.